lib/perl/OVH/Bastion/allowkeeper.inc

```diff
@@ -968,7 +968,7 @@ sub get_remote_accounts_from_realm {
     my %accounts;
     if (opendir(my $dh, "/home/allowkeeper/$sysaccount")) {
         while (my $filename = readdir($dh)) {
-            next if $filename !~ /allowed_([a-zA-Z0-9_-]+)\./;
+            next if $filename !~ /allowed_([a-zA-Z0-9._-]+)\.(ip|partial)/;
```

Comment:
You can have `.private` also, for personal accesses.
=> we might miss some realm tests around personal accesses!